### DISCUSSION TITLE
Tables side by side

### DISCUSSION THREAD

**Discussion by Radjoni** [dolanmiu/docx#1413] [Ideas / Requests]
Hello,

I just want to ask is it possible to show two tables side by side?

**Comment by ericfeldman-RCM** [comment]
I am trying to accomplish this as well. Not finding any way to do this currently.

**Comment by damian-garrido** [comment]
could you try to generate a "master" table with two columns and inside each column the respective tables?

**Comment by ericfeldman-RCM** [comment]
@damian-garrido That is exactly what I ended up figuring out, thanks for following up though! I think it could be helpful for others if this could be added to the table section in the usage doc.

**Comment by dolanmiu** [comment]
Yes, this is possible by using the approach outlined here:

https://www.youtube.com/watch?v=b6rBBzo9Uvg

The code for it is here:

https://github.com/dolanmiu/docx/blob/master/demo/77-side-by-side-tables.ts
